spec/controllers/questionnaire_controller_spec.rb

```diff
@@ -47,7 +47,7 @@
                                                   start_date: 1.week.ago.at_beginning_of_day,
                                                   person: mentor)
         responseobj = FactoryBot.create(:response, :invited, protocol_subscription: protocol_subscription,
-                                        open_from: 1.hour.ago)
+                                                             open_from: 1.hour.ago)
```

Comment:
I don't think rubocop will like this. 

Btw, you can have Rubymine's reformat fix this for you, if you have the alignment set up correctly (in settings, under editor, code style, ruby).

Reply:
my rubocop isn't complaining...